modules/deploy/pages/deployment-option/self-hosted/kubernetes/redpanda-cluster-recommendations.adoc

```diff
@@ -0,0 +1,91 @@
+= Redpanda Cluster Recommendations
+:description: A list of recommendations for deploying the Redpanda Helm chart in production.
+:tags: ["Kubernetes"]
+:page-aliases: deploy:deployment-option/self-hosted/kubernetes/kubernetes-best-practices.adoc
+
+This topic provides recommendations for deploying Redpanda in production.
+
+== Deploy at least three Pod replicas
+
+Redpanda Data recommends at least three Pod replicas (Redpanda brokers) to use as _seed servers_. Seed servers are used to bootstrap the gossip process for new brokers joining a cluster. When a new broker joins, it connects to the seed servers to find out the topology of the Redpanda cluster. A larger number of seed servers makes consensus more robust and minimizes the chance of unwanted clusters forming when brokers are restarted without any data.
```

Comment:
This file uses "Redpanda Data recommends..." but the [file before](https://deploy-preview-36--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements/) and the file after put Recommendations as a subheading. Is that intention, or could you format recommendations consistently in this file?